tests/restarting/from_7.1.0/VersionVectorDisableRestart-2.toml

```diff
@@ -0,0 +1,37 @@
+[[knobs]]
+enable_version_vector = false
+enable_version_vector_tlog_unicast = false
+
```

Comment:
"PROXY_USE_RESOLVER_PRIVATE_MUTATIONS" not disabled here.

Reply:
No need. `PROXY_USE_RESOLVER_PRIVATE_MUTATIONS` is only needed for VV, but can be independently used when VV is disabled.